sdk/anomalydetector/ai-anomaly-detector/test/anomalydetector.spec.ts

```diff
@@ -19,8 +19,7 @@ describe("AnomalyDetectorClient", () => {
   let recorder: Recorder;
   const apiKey = new AzureKeyCredential(testEnv.ANOMALY_DETECTOR_API_KEY);
 
-  beforeEach(function() {
-    // eslint-disable-next-line no-invalid-this
+  beforeEach(/** @this*/ function() {
```

Comment:
The usage of `@this` tag is to denote what "this" refers to in the function. So shouldnt this be `@this Mocha.Context`?

Reply:
Done.